mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +27,75 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Enlarge Image
+   */
+  isZoomable?: boolean;
+
+  /**
+   * Enlarge Image Styles
+   */
+  enlargeClassName?: string;
```

Comment:
i think it will be better to make all enlarge images the same for now, lets remove this prop to prohibit enlarge image view modifications outside of the component

Reply:
Removed!